grout-operator/roles/grout/meta/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+galaxy_info:
+  author: your name
```

Comment:
I wonder if we should fill this info with the Telco CI stuff

Reply:
Oh, nice catch, in fact this is not updated in the other operators. As I'm going to deprecate cnf-app-mac-operator next week, if you don't mind, I'll update this meta/main.yml file for all operators in my next change, so that I unblock the merge of this change. I'll note this in my pocket list. Thanks!